### DISCUSSION TITLE
ID3 tags in audio files should automatically be applied in the beatmap's metadata in the editor

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#21177] [Ideas] [answered]
In stable, importing an audio file that has ID3 tags present such as the song's title and artist gets automatically applied in the beatmap's respective metadata fields. In Lazer, this is absent, requiring the user to manually type out/copy the tags themselves.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/21189
